.github/workflows/pr-check.yaml

```diff
@@ -203,6 +203,9 @@ jobs:
       - name: Run unit tests
         run: yarn test:unit
 
+      - name: Run E2E smoke tests
+        run: yarn test:e2e:smoke
+
```

Comment:
AFAIK we can't put it the smoke e2e as a step

it needs to be a separate job so if it fails (as it seems it is not stable we can still merge PR)

Reply:
@odockal disagreement is about when to run it.

We said we wanted to be part of a PR check, not when we push to the main branch.

your PR https://github.com/containers/podman-desktop/pull/2774 trigger the workflow when there is a new commit in main branch
it should be a job triggered when user file a new PR

Reply:
so, in short, just add a new job at the end of pr-check.yaml file (a job, not a step)

Reply:
I see! thanks for explanation.